tests/unit/makefile

```diff
@@ -108,19 +108,19 @@ all: $(UNIT_TESTS)
 .SUFFIXES: .cpp .o
 .PHONY: all clean clean-exec
 
-unit_tests: $(SEQ_MAIN_OBJ) $(LIBTESTS_O) $(MFEM_LIB_FILE) $(CONFIG_MK) $(DATA_DIR)
+unit_tests: $(SEQ_MAIN_OBJ) $(LIBTESTS_O) $(MFEM_LIB_FILE)  $(DATA_DIR)
```

Comment:
Why remove this dependency?

Reply:
By mistake, reverted in 4f629c150d896dabc9ecfccdc46cd5486ddd2961.